string label
int64 track_id
geometry_msgs/Pose pose